07_cmd_lab/cmdlab.txt

```diff
@@ -3,11 +3,12 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
-  - your default shell?
-  - a list of all shells currently installed on the machine?
+ echo $0 - your default shell?
+  echo $SHELL  - a list of all shells currently installed on the machine?  cat /etc/shells       
 
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? details about the currnt user name passoword      What are the fields in each line?   login name, optional encrypted password, numerical user ID , numerical group ID, user name or comment field, user name directory, optional user command interpreter      
 
-3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")   etc SIZE       
```

Comment:
(Missing?) The correct answer is: `ls -S /etc`

The command is `ls` - which shows a list of files in a given directory
The option `-S` instructs ls to sort the entries by size
The argument `/etc` specifies the location to show

You can also use `ls -lS /etc` to get more details on each file, thus getting a chance to check the data is actually sorted correctly